src/routers/recipeRouter.ts

```diff
@@ -1,6 +1,9 @@
-const router = require("express").Router();
-const Recipe = require("../models/recipeModel");
-const auth = require("../middleware/auth");
+import express from "express";
+import { Request } from "express";
+import Recipe from "../models/recipeModel";
+import auth from "../middleware/auth";
+
+let router = express.Router();
```

Comment:
This should be `const`, not `let`, you don't change the value of this variable anywhere.